js/foundation.tabs.js

```diff
@@ -365,6 +371,7 @@ class Tabs {
       }
     }
 
+    $(window).off('popstate', this._checkDeepLink);
```

Comment:
I think we should. We should explicitly show that a piece of code is only required by an option.